boards/arm/mppt_2420_hpx/Kconfig.board

```diff
@@ -4,3 +4,19 @@
 config BOARD_MPPT_2420_HPX
 	bool "Libre Solar MPPT 2420 HPX"
 	depends on SOC_STM32G431XX
+
+if ADC_STM32
+
+config ADC_1
```

Comment:
Seems like this needs to be removed to get rid of the error. CONFIG_ADC_1 is already defined by Zephyr.